src/data/entity_attributes.ts

```diff
@@ -21,3 +21,32 @@ export const STATE_ATTRIBUTES = [
   "supported_features",
   "unit_of_measurement",
 ];
+
+export const TEMPERATURE_ATTRIBUTES = new Set([
+  "temperature",
+  "current_temperature",
+  "target_temperature",
+  "target_temp_temp",
+  "target_temp_high",
+]);
+
+export const ATTRIBUTES_UNITS: Record<string, string> = {
```

Comment:
Should we maybe make this a list per domain, and only do it for standard attributes of that domain? It feels a bit error prone like this?